torch/_inductor/codegen/cpp_wrapper_gpu.py

```diff
@@ -553,27 +559,34 @@ def generate_kernel_call(
                 device_index, call_args
             )
             kernel_var_name = self.generate_load_kernel_once(kernel_name, V.graph)
-
-            # args with value 1 are added into equal_to_1 and constants
-            # in triton_meta (in the Python codegen) which makes them
-            # inlined in the PTX and compiled CUBIN
-            arg_signatures = []
-            if (
-                triton_meta is not None
-                and triton_meta.get("configs")
-                and triton_meta.get("signature")
-            ):
-                equal_to_1 = triton_meta["configs"][0].equal_to_1
-                call_args = [
-                    arg for i, arg in enumerate(call_args) if i not in equal_to_1
-                ]
-                arg_types = [t for i, t in enumerate(arg_types) if i not in equal_to_1]
-                # extract the arg signatures from triton_meta
-                arg_signatures = triton_meta["signature"].values()
+            if triton_version_uses_attrs_dict():
                 arg_signatures = [
-                    v for i, v in enumerate(arg_signatures) if i not in equal_to_1
+                    val
+                    for key, val in triton_meta["signature"].items()
+                    if val != "constexpr"
                 ]
```

Comment:
This is the workaround I used to successfully launch: `DynamicShapesGpuWrapperGpuTests.test_consecutive_split_cumprod_xpu_dynamic_shapes_gpu_wrapper`
```suggestion
                ]
            signature = triton_meta["signature"]
            call_args = [call_arg for call_arg, arg_name in zip(call_args, signature) if signature[arg_name] != 'constexpr']
            arg_types = [arg_type for arg_type, arg_name in zip(arg_types, signature) if signature[arg_name] != 'constexpr']
```

Reply:
just to confirm, do you mean that `test_consecutive_split_cumprod_xpu_dynamic_shapes_gpu_wrapper` doesn't run on XPU prior to this change?

Reply:
> just to confirm, do you mean that `test_consecutive_split_cumprod_xpu_dynamic_shapes_gpu_wrapper` doesn't run on XPU prior to this change?

Yes, it didn't work. Before these changes I saw an error in accuracy. It was because the kernel expected 5 parameters, but 6 parameters were passed to it, and instead of the expected 5th parameter (called `s0`) an `xnumel==1` was passed.